### ISSUE TITLE
Fix missing form labels (a11y)

### ISSUE THREAD

**Issue by lmbateman** [camunda/camunda-modeler#4627]
### Problem you would like to solve

The following elements have missing form labels:

1. Help Center > Share your feedback: The “Describe your thoughts” text area doesn’t seem to have a label.
2. The reCAPTCHA box in the kapa-widget-container.
3. The search box in this form:

```
<form class="sc-hJkIYK zOWgo">
  <button type="button" title="Search" aria-label="Search through this list" data-test="expand-entity-search" class="sc-diYFot sc-kFFtEL lknmua wLKKB">
    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24">
      <path fill="var(--grey-darken-33)" d="M10.6 4c3.646 0 6.602 2.91 6.602 6.5a6.4 6.4 0 0 1-1.287 3.857L20 18.384 18.36 20l-4.128-4.07a6.64 6.64 0 0 1-3.631 1.072C6.955 17.002 4 14.092 4 10.5 4 6.91 6.955 4 10.6 4m0 2C8.06 6 6 8.015 6 10.5c0 2.487 2.06 4.502 4.6 4.502 2.542 0 4.602-2.015 4.602-4.501S13.142 6 10.6 6"></path>
    </svg>
  </button>
  <input type="text" placeholder="type to search..." aria-hidden="true" tabindex="-1" data-test="entity-search-input" class="sc-gxVrPM gqkxaL" value="">
</form>
```
See the attached screenshot for more information about missing form labels.

### Proposed solution

Make sure these elements have appropriate form labels.

### Alternatives considered

Leave elements unlabeled.

### Additional context

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/4031bdb5-ed4c-4f1e-b247-2aa934d43a94">
